packages/next/src/server/lib/router-utils/setup-dev.ts

```diff
@@ -203,6 +204,7 @@ async function startWatcher(opts: SetupOpts) {
       jsConfig,
       watch: true,
       env: process.env as Record<string, string>,
+      serverAddr: `127.0.0.1:${opts.port}`,
```

Comment:
```suggestion
      serverAddr: `${opts.hostname || 'localhost'}:${opts.port}`,
```
Should this be using hostname?

Reply:
No, it has to be an IP address, it's used in the edge runtime to fetch files from the dev server 

Kinda assuming there's never a case where the dev server is not reachable via `127.0.0.1` from the edge runtime